.github/workflows/release.yml

```
@@ -0,0 +1,40 @@
+    - name: Read Release Notes
+      id: release_notes
+      shell: pwsh
+      run: |
+        $notes = Get-Content "RELEASE_NOTES.md" -Raw
+        $notes = $notes -replace '\r\n', '%0A'
+        echo "content=$notes" >> $env:GITHUB_OUTPUT
+        
+    - name: Create Release
+      uses: softprops/action-gh-release@v2
+      with:
+        files: files: bin/Release/net6.0-windows/win-x64/publish/ScreensaverAuditor.exe
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 35: mapping values are not allowed in this context

(syntax-check)